maquetado/index.html

```diff
@@ -9,6 +9,12 @@
   </head>
   <body>
     <main class="main-project">
+      <header class="header-project">
```

Comment:
this header is still unnecesary

Reply:
Exactly, you are using an empty class i would remove this header